### PR TITLE
ENG-767 App builder title’s version using package.json version

### PR DESCRIPTION
**Pull request by jeffgo10**
this does not show version 6.2 but when package.json will now bump into 6.2 then so will be the version shown in the title

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **nicpuddu**

If they are not already assigned, you can assign the PR to them by writing `/assign @nicpuddu` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/env.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ function getClientEnvironment(publicUrl) {
         LEGACY_ADMINCONSOLE_INTEGRATION_ENABLED: process.env.LEGACY_ADMINCONSOLE_INTEGRATION_ENABLED === 'true',
         KEYCLOAK_ENABLED: process.env.KEYCLOAK_ENABLED === 'true',
         KEYCLOAK_JSON: process.env.KEYCLOAK_JSON || `${(process.env.DOMAIN || '')}/keycloak.json`,
+        APP_BUILDER_VERSION: process.env.npm_package_version || '',
```
Comment (ichalagashvili) [inline config/env.js:85]: Since it is a constants, for consistency, I think its better to be all uppercase

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/757#pullrequestreview-445635119" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/env.js (hunk 1 above)
Comment (gidesan) [reply config/env.js:85]: correct, I missed it

Comment (gidesan) [reply config/env.js:85]: nevermind, it's not changeable

**Pull request merged**